Explorer/Assets/DCL/PluginSystem/Global/Web3AuthenticationPlugin.cs

```diff
@@ -39,4 +69,12 @@ private void Login()
             web3Authenticator.LoginAsync(cancellationTokenSource.Token).Forget();
```

Comment:
Should not it be deleted from here as you launch `LoginAsync` from `AuthenticationScreenController`?

Reply:
I made a refactor that moves the login operation from the debug panel: `LoginFromDebugPanelSystem`
You have two options:
1. Direct login from a button
2. Open the authentication UI that executes the user flow